### PR TITLE
fix(ui): correct language usage re: "sync" vs. "scan"

### PR DESCRIPTION
**Pull request by TheCatLady**
#### Description

Since we do not sync Overseerr status/data back to services, these tasks are more accurately described as "scan" rather than "sync" jobs. (In the future, when syncing is added, it will be a separate job independent of the current "scan" jobs.)

**Note:** This PR updates the descriptions of Plex "sync" endpoints to reflect that they are actually triggering scans, however the endpoints themselves were not renamed to avoid yet another breaking API change.

#### To-Dos

- [x] Successful build `yarn build`
- [x] Translation keys `yarn i18n:extract`

#### Issues Fixed or Closed

N/A

### CODE REVIEW

File: server/job/schedule.ts [hunk 1]
```diff
@@ -32,63 +32,65 @@ export const startJobs = (): void => {
...
     cancelFn: () => jobSonarrSync.cancel(),
   });
 
-  // Run download sync
+  // Run download scan
   scheduledJobs.push({
-    id: 'download-sync',
-    name: 'Download Sync',
+    id: 'download-scan',
+    name: 'Download Scan',
```
Comment (sct) [inline server/job/schedule.ts:79]: This is actually in sync!

Comment (TheCatLady) [reply server/job/schedule.ts:79]: Doesn't "sync" infer it is two-way?  Don't we just scan and display the download status? 🤔

Comment (sct) [reply server/job/schedule.ts:79]: Not necessarily? It just means the state is synchronized. Which it is.

Comment (TheCatLady) [reply server/job/schedule.ts:79]: I suppose that's true.  But I still feel like it'd be clearer to have these current tasks which just fetch the current status of things be called "scans," and future jobs which do two-way synchronization to be called "syncs"?

(Just thinking ahead here.  IMO we should try to be deliberate and clear with language to minimize confusion around what these jobs actually do.)

**Review by sct — APPROVED** [review]

**Pull request merged**